libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -203,6 +208,17 @@ class AP_Airspeed
     // current primary sensor
     uint8_t primary;
     
+    // Rate of change of speed along X axis
+    float vel_dot;
+    float   _EAS_state;   // TECS airspeed
+    float   _integETAS_state;
+
+    int counter;
+    uint64_t _update_speed_last_usec;
+
+    // declares a 5point average filter using floats
+    AverageFilterFloat_Size5 vdot_filter;
```

Comment:
Why not make this a product of AHRS? As is you are having to retain the instance inside TECS as well, so now we are running 2 versions of the same filter.

Reply:
that should solve the rate problem too